robots/falcon_bravo7_description/urdf/falcon_bravo7.urdf

```diff
@@ -0,0 +1,452 @@
+                <inertial>
+                    <mass value="2.0"/>
+                    <origin xyz="0 0 0"/>
+                    <inertia ixx="0.01152" ixy="0.0" ixz="0.0" iyy="0.01152" iyz="0.0" izz="0.0218"/>
+                </inertial>
+                <visual>
+                    <origin rpy="0 0 0" xyz="0 0 0"/>
+                    <geometry>
+                        <mesh filename="package://example-robot-data/robots/falcon_bravo7_description/meshes/floater.stl"/>
+                    </geometry>
+                    <material name="orange"/>
+                </visual>
+                <collision>
+                    <origin rpy="0 0 0" xyz="0 0 0"/>
+                    <geometry>
+                        <mesh filename="package://example-robot-data/robots/falcon_bravo7_description/meshes/floater.stl"/>
+                    </geometry>
+                </collision>
+            </link>
+
+                    <joint name="falcon2floaterL" type="fixed">
+                        <parent link="falcon"/>
+                        <child link="floaterL"/>
+                        <origin rpy="-1.57 0 -1.57" xyz="-0.365 0.45 0.45"/>
+                    </joint>
+
+            <!--PROPELLERS-->
+            <link name="Z_propeller">
+                <visual>
+                    <geometry>
+                        <mesh filename="package://example-robot-data/robots/falcon_bravo7_description/meshes/propeller_rov.stl"/>
+                    </geometry>
+                    <material name="red"/>
+                </visual>
+            </link>
+
+            <link name="1_propeller">
+                <visual>
+                    <geometry>
+                        <mesh filename="package://example-robot-data/robots/falcon_bravo7_description/meshes/propeller_rov.stl"/>
+                    </geometry>
+                    <material name="red"/>
+                </visual>
+            </link>
+
+            <link name="2_propeller">
+                <visual>
+                    <geometry>
+                        <mesh filename="package://example-robot-data/robots/falcon_bravo7_description/meshes/propeller_rov.stl"/>
+                    </geometry>
+                    <material name="red"/>
+                </visual>
+            </link>
+
+            <link name="3_propeller">
+                <visual>
+                    <geometry>
+                        <mesh filename="package://example-robot-data/robots/falcon_bravo7_description/meshes/propeller_rov.stl"/>
+                    </geometry>
+                    <material name="red"/>
+                </visual>
+            </link>
+
+            <link name="4_propeller">
+                <visual>
+                    <geometry>
+                        <mesh filename="package://example-robot-data/robots/falcon_bravo7_description/meshes/propeller_rov.stl"/>
+                    </geometry>
+                    <material name="red"/>
+                </visual>
+            </link>
```

Comment:
Please change the `$number_propeller` name to something more meaningful, e.g, `FR_properller` (FR stands for forward-right)

Reply:
Done